templates/case/event/copy.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'layout-with-case-submenu.html.twig' %}
+
+{% trans_default_domain 'case_event' %}
+
+{% block title %}{% trans %}Copy case event to other cases{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Copy case event{% endtrans %}</h1>
+        </div>
+        <div class="col-auto">
+            <a class="btn btn-primary" href="{{ path('case_event_show', {id: case.id, caseEvent: case_event.id}) }}">{% trans %}Cancel{% endtrans %}</a>
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+    <main role="main" class="col-12 mt-3">
+        {{ form_start(form) }}
+        <div>
```

Comment:
Maybe we should tell the user that the case events are not actually copied, but a linked to other cases, i.e. changing a case event will change it on all (linked) cases.

Reply:
Added helper texting explaining that we link rather than copy.